kahuna/public/js/image/stream.js

```diff
@@ -0,0 +1,47 @@
+import Rx from 'rx';
+
+function imageStream(images_ = []) {
+
+    const images = new Set(images_);
```

Comment:
It's tempting to use an immutable set (from immutable-js?) here in the future, would make updates a lot more sexily functional.